src/Coordination/SnapshotableHashTable.h

```diff
@@ -327,13 +403,12 @@ class SnapshotableHashTable
         approximate_data_size = 0;
         for (auto & node : list)
         {
-            node.value.recalculateSize();
             approximate_data_size += node.key.size;
             approximate_data_size += node.value.sizeInBytes();
         }
     }
 
-    uint64_t keyArenaSize() const { return arena.allocatedBytes(); }
+    uint64_t keyArenaSize() const { return 0; }
```

Comment:
Why return 0? Do we still need this metric?

Reply:
we don't but I don't want to break backwards compatibility for now by removing this row from `mntr` command.